thunder/core/dtypes.py

```diff
@@ -154,12 +162,22 @@ class inexact(dtype):
 class floating(inexact):
     """Base class for the floating dtypes: bfloat16, float16, float32, float64."""
```

Comment:
what do you say to adding fp8 dtypes to this line?

Reply:
Oh yes! I missed it!